src/iedit.cpp

```diff
@@ -277,6 +277,24 @@ void iedit_disp_mod_vehicle_type_menu(struct descriptor_data *d)
   send_to_char(CH, "Set Compatible Vehicle Types: ^c%s^n\r\nEnter options (0 to quit): ", buf1);
 }
 
+/* reflex recorder linked skill */
+void iedit_disp_recorder_menu(struct descriptor_data *d)
+{
+  CLS(CH);
+  send_to_char( " 1) Edged Weapons\r\n"     " 2) Pole Arms\r\n"         " 3) Whips and flails\r\n"
+                " 4) Clubs\r\n"             " 5) Unarmed Combat\r\n"    " 6) Cyber Implants\r\n"
+                " 7) Pistols\r\n"           " 8) Rifles\r\n"            " 9) Shotguns\r\n"
+                "10) Assault Rifles\r\n"    "11) Submachine Guns\r\n"   "12) Mounted Gunnery\r\n"
+                "13) Tasers\r\n"            "14) Machine Guns\r\n"      "15) Assault Cannons\r\n"
+                "16) Projectiles\r\n"       "17) Throwing Weapons\r\n"  "18) Athletics\r\n"
+                "19) Stealth\r\n"           "20) Electronics\r\n"       "21) Biotech\r\n"
+                "22) Bike B/R\r\n"          "23) Car B/R\r\n"           "24) Drone B/R\r\n"
+                "25) Truck B/R\r\n"         "26) Pistol B/R\r\n"        "27) Shotgun B/R\r\n"
+                "28) Rifle B/R\r\n"         "29) Heavy Weapons B/R\r\n" "30) SMG B/R\r\n"
+                "31) Piloting Bike\r\n"     "32) Piloting Car\r\n"      "33) Piloting Truck\r\n"
+               "Enter skill for reflex recorder:\r\n", CH);
```

Comment:
I didn't see any bounds checking for this added in, what happens if someone enters -1 or 50? 

Also, it'd probably be better to have this draw directly from the skills constants-- use a for-loop to go over the skills from 1 to X, printing the number and name for each. Check out medit's skill assignment menu for an example of that.